packages/eslint-plugin/src/rules/no-emotion-css/index.ts

```diff
@@ -2,8 +2,8 @@ import type { Rule } from 'eslint';
 import type { ImportSpecifier, ImportDeclaration } from 'estree';
 
 import { buildImportDeclaration, buildNamedImport } from '../../utils/ast-to-string';
+import { COMPILED_IMPORT } from '../../utils/constants';
```

Comment:
Unrelated to the issue/change. However, I noticed that the constant was used several time. Hence, created a file for it